client/src/components/TrackList.vue

```diff
@@ -230,6 +263,29 @@ export default defineComponent({
               </template>
               <span>Default Type: {{ newTrackType }}</span>
             </v-tooltip>
+            <v-tooltip
+              open-delay="100"
+              bottom
+            >
+              <template #activator="{ on }">
+                <v-btn
+                  class="hover-show-child"
```

Comment:
No class with this name in the style section.

Reply:
Thoughts on reordering?  For whatever reason, this feels a lot nicer to me.  Maybe because the similar icon buttons are together.

![Screenshot from 2020-10-21 15-35-44](https://user-images.githubusercontent.com/4214172/96773821-1a861280-13b3-11eb-9ddc-4bf6c4a8e7e7.png)

Also, whatever you decide, could you make sure everything is `mx-2`?

Reply:
mx-2 feels like a bit much, are you okay with mr-2?
mx-2:
![image](https://user-images.githubusercontent.com/61746913/96777702-a51d4080-13b8-11eb-8b0e-1eb64a21f14b.png)
mr-2:
![image](https://user-images.githubusercontent.com/61746913/96777764-bebe8800-13b8-11eb-9edc-3c7f1562a3bb.png)

Reply:
Yeah, that's what I meant.  I still don't understand margin collapse rules.